package.json

```diff
@@ -22,6 +22,7 @@
 		"typescript": "4.7.4"
 	},
 	"dependencies": {
+		"events": "^3.3.0",
```

Comment:
Why do we need this dependency?

Reply:
These were attempts to run plugin. For some reason, I couldn't run it without this library)

Reply:
I think it is not good practice to commit PC-specific code to public repository....

Reply:
ok, I'll delete it in a couple of hours